src/ExpenseTrackerGrupo2/Business/Services/Interfaces/IUserService.cs

```diff
@@ -1,6 +1,7 @@
+using Microsoft.AspNetCore.Identity.Data;
```

Comment:
Can you tell me why are you using this import?

Reply:
its an unused import. im deleting

Reply:
Done